waku/node/v2/wakunode2.nim

```diff
@@ -324,4 +319,8 @@ when isMainModule:
       startMetricsServer(conf.metricsServerAddress,
         Port(conf.metricsServerPort + conf.portsShift))
 
+    # XXX: Is using discard here fine? Not sure if we want init to be async?
+    # Can also move this to the start proc, possibly wiser?
+    await result.subscribe(topic, handler)
```

Comment:
This call in general confuses me, I don' know why we are calling subscribe on result and we are also calling relay subscribe above.

Have you ran this and looked at logs and made sure subscribe is called accurately? Are both needed? I can't see the handler here either.

Reply:
@decanus please leave a comment when resolving, otherwise it isn't clear to me how something was resolved.